app-text/html2text/html2text-1.3.2a-r2.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="HTML to text converter"
+HOMEPAGE="http://www.mbayer.de/html2text/"
+SRC_URI="http://www.mbayer.de/html2text/downloads/${P}.tar.gz
+	http://www.mbayer.de/html2text/downloads/patch-utf8-${P}.diff
+	http://www.mbayer.de/html2text/downloads/patch-amd64-${P}.diff
+"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos"
+
+PATCHES=(
+	"${FILESDIR}/${P}-compiler.patch"
+	"${FILESDIR}/${P}-urlistream-get.patch"
+	"${DISTDIR}/patch-utf8-${P}.diff"
+	"${DISTDIR}/patch-amd64-${P}.diff"
+)
+
+src_prepare() {
+	default
+	gunzip ${PN}.1.gz html2textrc.5.gz || die
```

Comment:
To be honest, I think it would be better to remove PN from both.

Reply:
hehe, sure, i don't mind :)